src/index.js

```diff
@@ -1,12 +1,23 @@
 const chalk = require('chalk')
+const path = require('path')
 const fs = require('fs')
 const npm = require('npm')
 const Package = require('./package')
 
 const die = (message) => console.error(chalk.bold.red(message))
 const warn = (message) => console.warn(chalk.yellow(message))
 
-fs.readFile('package.json', 'utf-8', function(error, contents) {
+const argv = require('minimist')(process.argv.slice(2))
+
+let packagePath
```

Comment:
Maybe you could change the code to something like this:
```js
let packagePath = 'package.json';

if (argv.prefix && typeof argv.prefix === 'string') {
    packagePath = path.resolve(argv.prefix, 'package.json')
}
```
This way you can remove the else statement. Also, you could use a ternary operator and have everything in a single line. The change would be something like this:
```js
const packagePath = (argv.prefix && typeof argv.prefix === 'string') ? path.resolve(argv.prefix, 'package.json') : 'package.json'
```